applications/ray/workloads.tfvars

```diff
@@ -38,4 +38,5 @@ workload_identity_service_account = "ray-service-account"
 create_gcs_bucket               = true
 gcs_bucket                      = "ray-bucket-zydg"
 create_ray_cluster              = true
+ray_cluster_name                = "example-cluster"
```

Comment:
nit: can we call it something besides 'example-cluster'? how about just 'ray-cluster'?